### PR TITLE
Remove karma tests

### PR DESCRIPTION
**Pull request by becca-bailey**
- Removes older tests and config
- Updates test commands to run jest tests
- Converts a couple outstanding test files - node and victory native components

### CODE REVIEW

File: package-scripts.js [hunk 1]
```diff
@@ -14,24 +14,17 @@ module.exports = {
...
-      default: "mocha ./test/node"
+      default: "jest ./test/node"
     },
     test: {
-      cov: npsUtils.series.nps("build-package-libs", "karma.cov"),
-      dev: "karma start ./config/karma/karma.conf.dev.js",
-      watch: npsUtils.concurrent.nps("watch", "karma.watch"),
-      default: npsUtils.series.nps("build-package-libs", "karma")
+      watch: npsUtils.concurrent.nps("watch", "jest.watch"),
+      default: npsUtils.series.nps("build-package-libs", "jest.default")
```
Comment (becca-bailey) [inline package-scripts.js:27]: Do we still need to build packages before running tests in CI?

File: package.json [hunk 2]
```diff
@@ -128,10 +112,9 @@
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-hot-loader": "v2.0.0-alpha-4",
+    "react-test-renderer": "^16.13.1",
```
Comment (becca-bailey) [inline package.json:115]: This was necessary for `@testing-library/react-native`

File: package-scripts.js (hunk 1 above)
Comment (ryan-roemer) [reply package-scripts.js]: Yes, we do

File: test/node/setup.js [hunk 3]
```diff
@@ -1,2 +1 @@
-const { expect } = require("chai");
 global.expect = expect; // eslint-disable-line no-undef
```
Comment (scottrippey) [inline test/node/setup.js:1]: Pretty sure we should remove this too

File: package.json [hunk 4]
```diff
@@ -79,47 +80,28 @@
     "babel-plugin-lodash": "^3.1.2",
     "babel-plugin-module-resolver": "^4.1.0",
     "babel-preset-react-native": "^4.0.1",
-    "chai": "^3.5.0",
     "chromatic": "^5.0.0",
     "cross-env": "^7.0.3",
     "emotion-theming": "^10.0.27",
-    "enzyme": "^3.11.0",
```
Comment (scottrippey) [inline package.json:86]: 🔥 🎉

File: package.json [hunk 5]
```diff
@@ -79,47 +80,28 @@
...
-    "karma-benchmarkjs-reporter": "^1.0.0",
-    "karma-chrome-launcher": "^3.1.1",
-    "karma-coverage-istanbul-reporter": "^3.0.3",
-    "karma-firefox-launcher": "^2.1.2",
-    "karma-ie-launcher": "^1.0.0",
-    "karma-mocha": "^2.0.1",
-    "karma-safari-launcher": "^1.0.0",
-    "karma-sauce-launcher": "^4.1.5",
-    "karma-spec-reporter": "0.0.34",
-    "karma-webpack": "^5.0.0",
```
Comment (scottrippey) [inline package.json:116]: 🔥 🔥 🔥

**Review by scottrippey — APPROVED** [review]
Yay!

**Pull request merged**